src/renderer/components/uielements/assets/assetData/AssetData.tsx

```diff
@@ -22,17 +22,27 @@ export type Props = {
   price?: BaseAmount
   priceAsset?: PricePoolAsset
   size?: Styled.AssetDataSize
+  // `className` is needed by `styled components`
+  className?: string
 }
 
 export const AssetData: React.FC<Props> = (props): JSX.Element => {
-  const { asset, amount: assetAmount, noTicker = false, price = baseAmount(0), priceAsset, size = 'small' } = props
+  const {
+    asset,
+    amount: assetAmount,
+    noTicker = false,
+    price = baseAmount(0),
+    priceAsset,
+    size = 'small',
+    className = ''
```

Comment:
just a nitpick
usually `className` is optional and there is no need to use default value